authors/apps/authentication/views.py

```diff
@@ -67,7 +67,7 @@ def update(self, request, *args, **kwargs):
         serializer = self.serializer_class(
             request.user, data=serializer_data, partial=True
         )
-        serializer.is_valid(raise_exception=True)
+        # serializer.is_valid(raise_exception=True)
```

Comment:
@felixkiryowa Why is this commented out? Doesn't this check if the data is valid, if not raise an exception?